Update Submit verification Caught Profile page up with rest of pages + Address autocomplete

baloveless opened the pull request:
- Changed submit verify to be easier to understand
- Modified formUtil to accommodate profile page use
    - Added param for passing in a swap state function (i.e. switch from seeing to editting user information)
    - Param for passing in address autocomplete function (optional functions could be changed to being passed in by JSON)
- Implemented simple address autocomplete. Possibly overly simple because of the use of datalist html item that excludes anything that isn't a close match. Usually only shows one option. 
- removed some of the button options I added as they were redundant.

baloveless committed:
hold changes

baloveless committed:
Merge branch 'development' into submit-verify

baloveless committed:
made submit verification easier to understand.

baloveless committed:
hide .env

baloveless committed:
removed env

baloveless committed:
Have delay working for address auto complete

baloveless committed:
Profile page is caught up but has issues
* There is a failure on submit, 500 error.

baloveless committed:
Done with profile transition

baloveless committed:
removed bad css

baloveless committed:
fixed tests

mackfoggia commented on frontend/src/util/config.ts:
```diff
@@ -4,9 +4,9 @@ import dotenv from "dotenv"
+const server = "http://localhost:5000";
 // const server = 'http://34.82.220.171';
-const server = 'http://appstatus.brownhope.org';
+//const server = 'http://appstatus.brownhope.org';
```
```suggestion
const server = 'http://appstatus.brownhope.org';
```

baloveless committed:
Autocomplete, comments, and css improved.
- Autocomplete works + looks better
- added comments describing most functions.
- Added height controls for the nav, current page, and footer to config

baloveless committed:
almost forgot config again

baloveless commented:
Made the autocomplete better, now it shows more options and is easier to understand than before. Also updated all the comments to make the whole thing easier to understand.

baloveless committed:
Added an onblur effect to remove drop down

baloveless committed:
Got unexpected behavior, removing last change.
